tron/bin/check_tron_jobs.py

```diff
@@ -428,6 +440,10 @@ def main():
     error_code = 0
     global _run_interval
     _run_interval = args.run_interval
+
+    global _skip_superregion
+    _skip_superregion = args.skip_failed_logging_for_superregion
```

Comment:
hmm, interesting pattern that previous-CI left here - not quite sure I'm a fan, but I guess we can continue adding onto this for now until we get around to refactoring this whole thing